DOC: update user manual for functions that ~moved from shapely.ops to shapely (#2030)
shapely/shapely: docs/manual.rst

   <POLYGON ((6 5, 2 1, 1 2, 1 5, 6 5))>]

Nearest points/shortest line
----------------------------

The :func:`~shapely.shortest_line` function calculates the shortest line between a pair
of geometries.

.. function:: shapely.shortest_line(geom1, geom2)

   Returns a tuple of shortest line between the input geometries. The points are
   returned in the same order as the input geometries.

>    `New in version 1.4.0`.

Reviewer: ```suggestion
   `New in version 2.0`.
```

.. code-block:: pycon

  >>> from shapely import shortest_line
  >>> triangle = Polygon([(0, 0), (1, 0), (0.5, 1), (0, 0)])
  >>> square = Polygon([(0, 2), (1, 2), (1, 3), (0, 3), (0, 2)])
  >>> shortest_line(triangle, square)
  <LINESTRING (0.5 1, 0.5 2)>

> Note that the nearest points may not be existing vertices in the geometries.

Reviewer: ```suggestion
Note that the shortest line may not connect to vertices of the input geometries.
```

Splitting
---------

The :func:`~shapely.ops.split` function in `shapely.ops` splits a geometry by another
geometry.

.. function:: shapely.ops.split(geom, splitter)

   Splits a geometry by another geometry and returns a collection of geometries. This
   function is the theoretical opposite of the union of the split geometry parts. If the
   splitter does not split the geometry, a collection with a single geometry equal to
   the input geometry is returned.

   The function supports:

   * Splitting a (Multi)LineString by a (Multi)Point or (Multi)LineString or (Multi)Polygon boundary

   * Splitting a (Multi)Polygon by a LineString

   It may be convenient to snap the splitter with low tolerance to the geometry. For
   example in the case of splitting a line by a point, the point must be exactly on the
   line, for the line to be correctly split. When splitting a line by a polygon, the
   boundary of the polygon is used for the operation. When splitting a line by another
   line, a ValueError is raised if the two overlap at some segment.

> 
>    `New in version 1.6.0`

Reviewer: ```suggestion
```

1.6 is ancient history, I think we can safely remove this. Perhaps we remove all such references to releases before 2.0?
